lxd/instance/drivers/driver_qemu.go

```diff
@@ -6857,6 +6857,39 @@ func (d *qemu) migrateSendLive(pool storagePools.Pool, clusterMoveSourceName str
 		return err
 	}
 
```

Comment:
Please remove the "incus issue" reference in the commit message as not relevant.